index.html

```diff
@@ -35,7 +37,174 @@ <h1>I'm Safar Ali <br> A thriving <br> Software Developer </h1>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
+            <div class="buttons">
+                <button>See Project</button>
+            </div>
+        </section>
+            
+        <section class="page-3">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-2.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Multi-Post Stories</h1>
+            </div>
+            <div class="year">
+                <p>Safar</p>
+                <p>Backend Dev</p>
+                <p>2015</p>
+            </div>
+            <div class="para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
+            <div class="buttons">
+                <button>See Project</button>
+            </div>
+        </section>
+             
+        <section class="page-4">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-3.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Tonic</h1>
+            </div>
+            <div class="year">
+                <p>Safar</p>
+                <p>Backend Dev</p>
+                <p>2015</p>
+            </div>
+            <div class="para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
+            <div class="buttons">
+                <button>See Project</button>
+            </div>
+        </section>
+
+        <section class="page-5">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-4.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Multi-Post Stories</h1>
+            </div>
+            <div class="year">
+                <p>Safar</p>
+                <p>Backend Dev</p>
+                <p>2015</p>
+            </div>
+            <div class="para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
+            <div class="buttons">
+                <button>See Project</button>
+            </div>
+        </section>
+
+    </main>
+
+    <!-- About Us page -->
+
+    <section class="about-us">
+        <div class="title">
+            <h1>About <br> Myself </h1>
```

Comment:
- [ ] As I already mentioned before, having only 1 `h1` is important for semantics and **SEO** reasons! Kindly change this heading 🤓

Reply:
> * [ ]  As I already mentioned before, having only 1 `h1` is important for semantics and **SEO** reasons! Kindly change this heading 🤓

- [ ] You did great on changing this h1 to `hx`, however, there are still some changes you will need to work on before you go onto the next step, let's see what they are.
- [ ] Compared to the original design, your page needs some improvement on this section too, kindly fix them all to match the Figma design, remember to stick to the design with colors, padding, margins, font-size, line-height, and font-family, 
<img width="316" alt="abt yyy" src="https://user-images.githubusercontent.com/79058364/153720563-bba09ce0-0972-47ad-a7a1-e691038cb512.png">
Looks good but not better 😄 , please adjust your styling to match this design below ⏬ 

- Figma design:
<img width="253" alt="abt ffffffff" src="https://user-images.githubusercontent.com/79058364/153720608-4ae2e9aa-cce8-48ec-bcb4-51159760d774.png">